src/main/java/executor/service/objectfactory/ClassScannerUtil.java

```diff
@@ -2,7 +2,7 @@
 
 import org.reflections.Reflections;
 
-public class ClassScannerUtil {
+class ClassScannerUtil {
 
   private static Reflections CLASS_SCANNER = new Reflections("/src/java/executor/service/");
```

Comment:
This is not best practice.

Reply:
So let's some think, do you need any where except ObjectFactory classscanner?